### PR TITLE
Successful networked player flipping

### LINKED ISSUE #12: Rotate Bird Icon on All Clients
Ensure rotation state of bird is synchronized from owner to other clients via a `NetworkVariable`.

Make sure it updates on other clients as expected.

### PR DESCRIPTION
**Pull request by swiimii**
# Description

Add code which syncs player direction flipping over the network.

Please include a summary of the change and which issue is fixed or what was added to the project.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that demonstrate the new feature or bugfix
- [ ] New and existing unit and integrations tests pass locally with my changes

### CODE REVIEW

File: Packages/manifest.json [hunk 1]
```diff
@@ -21,15 +21,18 @@
     "com.unity.ide.visualstudio": "2.0.17",
     "com.unity.ide.vscode": "1.2.5",
     "com.unity.inputsystem": "1.4.4",
+    "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main",
```
Comment (nicholas-maltbie) [inline Packages/manifest.json:24]: Revert changes so in sorted order again.

Comment (swiimii) [reply Packages/manifest.json:24]: I'm having issues doing this in Git; let's handle it later.

**Comment by nicholas-maltbie** [comment]
Closes Issue #12

**Comment by swiimii** [comment]
#12

**Review by nicholas-maltbie — APPROVED** [review]

**Pull request merged**